### PR TITLE
wireframe implementation

### PR DESCRIPTION
**Pull request by mpontus**

### CODE REVIEW

**Comment by mpontus** [comment]
Thus branch currently features an adaptive wireframe layout built with [Rebass](https://github.com/rebassjs/rebass): https://deploy-preview-7--kf-web.netlify.app/

Next is to replace Rebass with Fabric.

**Review by onnovisser — COMMENTED** [review]
Nice. Looking good. One thing I'd add is [gatsby-plugin-styled-components](https://www.gatsbyjs.com/plugins/gatsby-plugin-styled-components/), so the styles are pre-rendered.

**Review by onnovisser — APPROVED** [review]
👍

File: src/pages/index.js [hunk 1]
```diff
@@ -81,47 +81,41 @@ const badgeStyle = {
...
+    description: `A great place to get started if you're new to web development. Designed to guide you through setting up your first Gatsby site.`,
+    color: `#E95800`,
   },
   {
-    text: "How to Guides",
-    url: "https://www.gatsbyjs.com/docs/how-to/",
-    description:
-      "Practical step-by-step guides to help you achieve a specific goal. Most useful when you're trying to get something done.",
-    color: "#1099A8",
+    text: `How to Guides`,
```
Comment (lucasvo) [inline src/pages/index.js:90]: @mpontus This should probably be removed?

**Review by lucasvo — CHANGES_REQUESTED** [review]

**Pull request merged**